frameworks/svelte/jsdocs/AdvancedImage.tsx

```diff
@@ -0,0 +1,33 @@
+/**

```

Comment:
@newmaniese Is there a reason this was added to the PR?

Reply:
It can be removed, it was just a part of the testing/packaging scripts. I wasn't sure if you wanted those committed. I can distill this to the changes required.

Reply:
Ahh in that case it can be removed as well:)